### DISCUSSION TITLE
Add Project dropdown remains empty until we search the project while adding project in Repo

### DISCUSSION THREAD

**Discussion by anshulsahni** [community/community#16497] [Projects and Issues]
TLDR; The Add Project dropdown in Project (Beta) at repository level shows nothing unless I search the project, that's very confusing, for a very long time I was confused that when I already have a project at my profile level then why isn't the project visible there.

**1. I navigate to Projects (Beta) section in my repository 👇🏼** 
<img width="836" alt="image" src="https://user-images.githubusercontent.com/4351432/168220631-598c50f9-b053-4032-aa09-97bc4ef6a11c.png">

**2. Next, I click on Add Project button, and a searchable dropdown opens up, which says `No recent projects`**
<img width="440" alt="image" src="https://user-images.githubusercontent.com/4351432/168220781-f1c0c53b-596f-4bac-9b92-6da14e1454d6.png">

**3. But when I searched my project with it's name, it appeared**
<img width="473" alt="image" src="https://user-images.githubusercontent.com/4351432/168220888-5e5debb1-6e3e-4da2-94b5-d177ca112445.png">

This is bizarre behavior, since the user when he/she reaches the add project panel would like to see the existing project, unless projects are more than a certain number (let's say 5), I would want to see all my projects listed there
